src/components/enterprise-user-subsidy/offers/data/reducer.js

```diff
@@ -5,7 +5,7 @@ import {
 } from './constants';
 
 export const initialOfferState = {
-  loading: true,
+  loading: false,
```

Comment:
The `FETCH_OFFERS_REQUEST` action will set `loading: true` when the `fetchOffers()` function is called.